src/openrct2-ui/interface/Widget.h

```diff
@@ -19,7 +19,10 @@ ImageId GetColourButtonImage(colour_t colour);
 Widget* GetWidgetByIndex(const WindowBase& w, WidgetIndex widgetIndex);
 
 constexpr auto kWidgetsEnd = Widget{ WindowWidgetType::Last, 0, 0, 0, 0, 0, 0, 0 };
+constexpr uint32_t kWidgetContentEmpty = 0xFFFF'FFFF;
```

Comment:
We don’t use this notation anywhere in the codebase. We could of course decide to change it, but that would something for the team chat, I’d say.

Reply:
You mean the apostrophe to separate digits? It's a C++17 thing, I believe. Considering we started out with C, it's not that surprising the codebase doesn't use it yet.

If you believe its introduction warrants discussion, please raise it in the team chat, as suggested.

Reply:
Yes, I believe it does.